src/json.hpp.re2c

```diff
@@ -6906,17 +7178,40 @@ class basic_json
 
         @throw std::range_error if passed value is out of range
         */
-        long double get_number() const
+        void get_number(basic_json& result) const
```

Comment:
In the documentation, there is still the `@return` part which makes no sense any more as the function is `void`. Could you please adjust the documentation to described the side effect of the function?